src/annotator/index.js

```diff
@@ -51,7 +51,14 @@ function init() {
   const sidebar = !config.subFrameIdentifier
     ? new Sidebar(document.body, eventBus, guest, config)
     : null;
-  const notebook = new Notebook(document.body, eventBus, config);
+  // Clear `annotations` value from the notebook's config to prevent direct-linked
+  // annotations from filtering the threads.
+  //
+  // TODO: Refactor configFrom() so it can export application specific configs
```

Comment:
The use of TODO in committed production code has been discouraged over time, but I actually think this is an appropriate usage. Can you add a link to the ticket you opened tracking this to the comment?

I've also added a note to talk about TODO/FIXME comments in code at our next frontend sync, again.